back/engines/commercial/analysis/app/lib/analysis/LLM/azure_open_ai.rb

```diff
@@ -5,11 +5,15 @@
 
 module Analysis
   module LLM
-    class OpenAIGPT < Base
+    class AzureOpenAI < Base
       def initialize(**params)
         super
+
         @client = OpenAI::Client.new(**{
-          access_token: ENV.fetch('OPENAI_API_KEY'),
+          access_token: ENV.fetch('AZURE_OPENAI_API_KEY'),
+          uri_base: [ENV.fetch('AZURE_OPENAI_URI'), '/openai/deployments/', azure_deployment_name].join,
```

Comment:
Is there a different `AZURE_OPENAI_URI` per cluster (so that the data is processed in the same region)?

Reply:
Yes, and I already added the ENV varialbles to all clusters.

We have these regions:
![image](https://github.com/CitizenLabDotCo/citizenlab/assets/329308/6054fe5f-186c-4bc6-aba4-8e914c47a266)